hoomd/md/external/field.py

```diff
@@ -48,7 +50,7 @@ class Periodic(Field):
 
     .. math::
 
-       V(\\vec{r}) = A * \\tanh\\left[\\frac{1}{2 \\pi p w} \\cos\\left(
+       V(\\vec{r}) = A \\tanh\\left[\\frac{1}{2 \\pi p w} \\cos\\left(
```

Comment:
I removed stars in a few places where the LaTeX rendering is clear that there is an implied multiplication operation, especially because the star renders very prominently and makes me question if there is some kind of unusual product operation being used.